app/timetables/tests/test_models.py

```diff
@@ -65,3 +65,63 @@ def test_meal_end_time_same_with_start_time_cannot_be_saved(self):
         )
 
         self.assertRaises(ValidationError, meal.save)
+
+
+class TimetableTest(TestCase):
+    """Tests the Timetable model."""
+
+    def setUp(self):
+        Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=1,
+            current_cycle_day=2,
```

Comment:
@andela-kanyanwu I checked the doc and the above is still valid. Unless you want  to add some context to what you are driving at.